.gitlab-ci.yml

```diff
@@ -568,8 +569,9 @@ publish_community_operators:
       - if: $CI_COMMIT_TAG
         when: manual
       - when: never
-  needs:
-    - "submit_preflight_redhat_image"
+  # Allow the job to be run manually even if submit_preflight_redhat_image fails, so that even if preflight submission is done locally this job can be used to open the PRs
+  # needs:
```

Comment:
The functionality would be the same, but rather than commenting these lines out, we can allow the `submit_preflight_redhat_image` job to fail and still allow this job to be triggered.

Reply:
oops, idk how to comment on the collapsed lines, but I think we can uncomment out the `needs` line and add 

`  allow_failure: true`

after line 564

Reply:
i like this suggestion, it has the benefit of not marking the whole pipeline as failed if this fails